src/components/Video/Video.component.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Styled from './styled';
+
+const Video = ({ etag, snippet }) => {
+  return (
+    <Styled.Container key={etag}>
```

Comment:
if you're uncertain of the uniqueness of etag value, you might use the array index as well